src/PhpSpec/Message/Fatal.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Message;
+
+class Fatal implements MessageInterface {
+    private $currentExample;
+
+    public function setMessage($currentExample)
```

Comment:
My thoughts were to allow other developers to build messages for any further integration (e.g a Warning Message, Error Message....)